tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -122,3 +122,46 @@ Feature: spaces management
       | David | viewer  |
       | Edith | viewer  |
     And "Alice" logs out
+
+
+  Scenario: admin user can manage the spaces created by other space admin user
+    Given "Admin" creates following users using API
+      | id    |
+      | Alice |
+      | Brian |
+      | Carol |
+    And "Admin" assigns following roles to the users using API
+      | id    | role        |
+      | Alice | Admin       |
+      | Brian | Space Admin |
+      | Carol | Space Admin |
+    When "Brian" logs in
```

Comment:
does `Brian` have to login before creating space using API

Reply:
api request is done through token so user has to login for the token.